src/helpers/player.js

```diff
@@ -185,6 +185,18 @@ const playlist = (collection, mediaType, contentLanguage, uiLanguage) => {
   };
 };
 
+const playlistFromUnits = (collection, mediaType, contentLanguage, uiLanguage) => {
+  const items = collection.content_units
+    .map(x => playableItem(x, mediaType, uiLanguage, contentLanguage))
+    .filter(item => !!item.unit)
```

Comment:
You can check `item.unit` inside of `playableItem()` and eliminate second `.map()`.
Anyway it is better to `.filter` first and to `.map()` shorter lists.

Reply:
Was think that stream functions like map/filter ... are use one interaction. So its not affect on performance ?

Reply:
That's not true, but who cares